node/tools/src/config.rs

```diff
@@ -210,6 +214,7 @@ impl Configs {
                         self.app.max_payload_size,
                     )),
                 }),
+            attester: None,
```

Comment:
if it is supposed to be unused for now, then don't add the attester_secret_key to the config at all.

Reply:
I forgot to delete it sry. I'll move all this unused code to a new branch beacuse it would be useful